sources/packages/web/src/components/common/AnnouncementBanner.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <v-alert
+    :type="BannerTypes.Warning"
+    variant="outlined"
+    icon="fa:fa fa-triangle-exclamation"
+    class="sims-banner"
+  >
+    <template #title>
+      <div class="mb-5">System Announcements</div>
+    </template>
+    <v-row>
+      <v-col>
+        <div
+          v-for="(announcement, index) in relevantAnnouncements"
+          :key="index"
+        >
+          <div class="label-value-normal">
+            <b>{{ announcement.messageTitle }}</b> -
```

Comment:
We can use the class `font-bold` here instead of `<b>`.

Reply:
@katrina-aot-git To be clear, we can use the `<strong>{{ announcement.messageTitle }}</strong>` instead of the `<b>`.
We have a simple CSS that will be applied to `strong`.
https://github.com/bcgov/SIMS/blob/f1bb49e2c447803197e9c517f0a7599075d40993/sources/packages/web/src/assets/css/base.scss#L14-L21

Reply:
reworked

Reply:
I cannot see this change.

Reply:
Resolving this since there is another comment open.